ci-operator/step-registry/ibmcloud/deprovision/resourcegroup/ibmcloud-deprovision-resourcegroup-commands.sh

```diff
@@ -49,6 +49,13 @@ function run_command_with_retries() {
 
 ibmcloud_login
 
+#OCP-60816 - [IPI-on-IBMCloud] install cluster under BYON with a different resource group
+#after destroyed the cluster, check the pre resource group is existed.
+if [ -s "${SHARED_DIR}/ibmcloud_cluster_resource_group" ]; then
+    resource_group=$(cat "${SHARED_DIR}/ibmcloud_cluster_resource_group")
+    ${IBMCLOUD_CLI} resource group ${resource_group}
```

Comment:
Beside we need to ensure the cluster resource exists, also need to check the resource group is empty, right?

Reply:
since the following will delete the resource group, if it is not empty, destroying it will be failed.